torch/_functorch/pyfunctorch.py

```diff
@@ -122,12 +123,28 @@ def prev_grad_mode(self):
         return self._cptr.prevGradMode()
 
 
+class FunctionalizeInterpreter(FuncTorchInterpreter):
+    def __init__(self, cdata: CInterpreter):
+        assert cdata.key() == TransformType.Functionalize
+        self._cdata = cdata
+        self._cptr = CFunctionalizeInterpreterPtr(cdata)
+
+    def process(self, op, args, kwargs):
+        kernel = op.functorch_table[TransformType.Functionalize]
+        return kernel(self, *args, **kwargs)
+
+    def functionalizeAddBackViews(self):
+        return self._cptr.functionalizeAddBackViews()
```

Comment:
nit: functionalize_add_back_views (FuncTorchInterpreter subclasses do snake_case, while the CInterpreterPtrs do camelCase)

Reply:
done